src/main/java/com/parkingapp/parkingservice/domain/common/Location.java

```diff
@@ -0,0 +1,13 @@
+package com.parkingapp.parkingservice.domain.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+public class Location {
```

Comment:
I'm wondering if we should validate that the values for latitud and longitud are in a valid range?
Validate on constructor and setter (if any)
What do you think?

Reply:
Added validation to handle invalid params for location. good call 💡